src/axios.ts

```diff
@@ -193,6 +200,30 @@ export function multinetAxiosInstance(config: AxiosRequestConfig): MultinetAxios
     return this.get(`/workspaces/${workspace}/tables/${table}/metadata`);
   };
 
+  Proto.uploadNetwork = async function(workspace: string, network: string, options: FileUploadOptionsSpec, config?: AxiosRequestConfig): Promise<AxiosResponse<Array<{}>>> {
+    const headers = config ? config.headers : undefined;
+    const params = config ? config.params : undefined;
+    const { type, data, url } = options;
+    const s3ffClient = new S3FileFieldClient({
+      baseUrl: `${url}/api/s3-upload/`,
```

Comment:
Shouldn't `url` be the same as the `MultinetAxiosInstance`'s `baseUrl` property?

If possible, we shouldn't make the user of this function supply this value, since the `s3-upload` path is part of the same API the rest of this interface is serving.

Reply:
Great point, it is!

See a344b21